README.md

```diff
@@ -1 +1,253 @@
+    ```C
+    volatile RAIL_RxChannelHoppingConfigMultiMode_t duty_cycle_multi_mode_config = {
+        .timingSense = 1600,
+        .preambleSense = 2200,
+        .syncDetect = DUTY_CYCLY_SYNC_TIMEOUT,
+        .timingReSense = 1600,
+        .status = 0,
+    }; 
+    /// Config for the correct timing of the dutycycle API
+    RAIL_RxDutyCycleConfig_t duty_cycle_config = {
+      .delay = ((uint32_t) DUTY_CYCLE_OFF_TIME),
+      .delayMode = RAIL_RX_CHANNEL_HOPPING_DELAY_MODE_STATIC,
+      .mode = RAIL_RX_CHANNEL_HOPPING_MODE_MULTI_SENSE,
+      .parameter =  (uint32_t) (void *) &duty_cycle_multi_mode_config
+    //  .parameter = ((uint32_t) DUTY_CYCLE_ON_TIME)
+    };
+    ```
+-  Please find the ***sl_duty_utilicy.c*** file in the path ***gecko_sdk_3.2.1/app/flex/component/rail/sl_duty_cycle_core*** and modify the funcation of calculating preamble bit length as following. Please select the **Make a Copy** when you try to change this file.  
+    ```C
+    uint16_t calculate_preamble_bit_length_from_time(const uint32_t bit_rate, RAIL_RxDutyCycleConfig_t * duty_cycle_config)
+    {
+      float on_time = 0;
+      float off_time = 0;
+      float preamble_time = 0;
+      float preamle_bit_length = 0;
+
+      on_time = DUTY_CYCLE_ON_TIME;
+      off_time = duty_cycle_config->delay;
+      preamble_time = ((float)(PREAMBLE_PATTERN_LENGTH * PREAMBLE_PATTERN * PREAMBLE_OVERSAMPLING) * U_SEC) / bit_rate;
+
+      app_assert(preamble_time < on_time, "Please modify the on time according to the bitrate!\n");
+
+      while (1) {
+        preamble_time = (off_time + (2 * on_time)) / 1000000;
+        preamle_bit_length = (preamble_time * bit_rate);
+        if (preamle_bit_length <= 50000) {
+          break;
+        }
+        off_time = off_time - on_time;
+      }
+
+      if (((uint32_t)off_time) != DUTY_CYCLE_OFF_TIME) {
+        app_log_warning("Duty Cycle Off time was changed to ensure stable working\n");
+      }
+
+      return preamle_bit_length;
+    }
+    ```
+### Modify the Radio PHY configuations
+Please download the ***rail_config.c*** and ***rail_config.h*** file as the following link, and replace the original file is located in **Autogen** folder, rebuild the project and flash to the device.   
+[Optimized 9p6k radio configuation](radio_configuations/optimized_9p6k_configuation)  
+
+**Note**: Those two files will be overwritten if you change anythings in radio generator.  
+
+## Test conclusion
+### Power Consumption
+After optimzing the EM2 current and radio PHY. **Iavg = 19.036 uA**  
+<img src="images/optimized power consumption.gif">
+
+Below table is the comparison of power consumption between the sample project and the optimized project.
+|   project   |sample peoject| optimized project| 
+|:----:| :-----------:| :----------------:| 
+|**average current**| 40.674 uA| 19.036 uA |  
+
+**Conclusion:** The power consumption reduce by **53%** to **19.036 uA** after optimizing. 
```

Comment:
The average current consumption of the sample project with optimized radio configuration is 19.04uA, which is much better than the average current consumption of the original sample project. From the table, the average current consumption reduces 53%, which is a very huge improvement.